src/components/button/button-theme.scss

```diff
@@ -5,7 +5,7 @@ a.md-button.md-THEME_NAME-theme,
     &:hover {
       background-color: '{{background-500-0.2}}';
     }
-    &.md-focused {
+    &.md-focused:not(.md-no-focus) {
```

Comment:
I'm not getting what you exactly mean. You mean we should wrap that as followed:
```
    &.md-focused {
      &:not(md-no-focus) {
        background-color: '{{background-500-0.2}}';

      }
    }
```

Because I think this will add an empty selector, which is not used. (`&.md-focused`)